markdown-it-decksetter.js

```diff
@@ -109,11 +107,11 @@ module.exports = function containerPlugin (md, name, options) {
         continue
       }
 
-      for (position = start + 1; position <= max; position++) {
+      /*for (position = start + 1; position <= max; position++) {
```

Comment:
Commenting this out caused the all of closing `</article>` elements to appear. Here it is without this commented (and with my comment debug):

```
<!-- -->
<article class="slides">
<p>this is the first slide</p>
<p>^ here are some notes</p>
</article>
<h1>the second slide</h1>
<p>this is the second slide</p>
<p>^ here are some more notes</p>
<!-- -->
<article class="slides">
<h1>the third slide</h1>
<p>hi i am a third slide</p>
<p>^ whoa even more notes</p>
</article>
```